BricksAndHearts.UnitTests/ControllerTests/Landlord/LandlordControllerTests.cs

```diff
@@ -37,4 +43,65 @@ public async void ApproveCharter_CallsApproveLandlord()
         // Assert
         A.CallTo(() => landlordService.ApproveLandlord(landlord.Id, adminUser)).MustHaveHappened();
     }
+
+    [Fact]
+    public void ViewProperties_CalledByAdmin_CanReturnOtherUsersPropertyList()
+    {
+        // Arrange
+        var adminUser = CreateAdminUser();
+        var fakeLandlordService = A.Fake<ILandlordService>();
+        var fakePropertyService = A.Fake<IPropertyService>();
+        var _underTest =
+            new LandlordController(A.Fake<ILogger<LandlordController>>(), fakeLandlordService,
+                fakePropertyService, null!);

```

Comment:
Hopefully this stuff will already be setup in the class if you rebase, so you won't have to repeat it